composer.json

```diff
@@ -50,6 +50,7 @@
         "liip/imagine-bundle": "1.3.0",
         "imagine/imagine": "0.6.2",
         "monolog/monolog": "1.22.0",
+        "ocramius/proxy-manager": "2.0.4",
```

Comment:
It is a dependency of one of our doctrine packages (don't remember which one). I suppose JJ fixed it to the last version.
@jjanvier beware, I think I remember the version of the bundle is different (maybe even not needed) depending of PHP version.